onnxruntime/core/graph/graph.cc

```diff
@@ -484,6 +484,7 @@ Status Node::UpdateInputArgCount() {
 
   if (total_arg_count < 0 || static_cast<size_t>(total_arg_count) != definitions_.input_defs.size()) {
     return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL,
+                           "This is an invalid model. "
```

Comment:
What's the difference between invalid model and invalid graph?

Reply:
invalid graph -> graph is acyclic, etc. Usually has StatusCode = INVALID_GRAPH while constructing the Status object

invalid model -> graph itself is valid, but has problems like duplicate node names, et.c.

Reply:
Edges are part of the graph, no? So, why is the graph valid in this case?

Reply:
Because there is nothing wrong with the edges themselves. Just that they are bound to two different types. I am open to not go with this finer differentiation. How about we just stick the "...invalid model" ?